tensorbay/label/label_polyline.py

```diff
@@ -122,8 +137,9 @@ class LabeledPolyline2D(_LabelBase, Polyline2D):  # pylint: disable=too-many-anc
     _T = TypeVar("_T", bound="LabeledPolyline2D")
 
     _repr_type = ReprType.SEQUENCE
-    _repr_attrs = _LabelBase._repr_attrs
+    _repr_attrs = ("beizer_point_types",) + _LabelBase._repr_attrs
     _attrs_base: Polyline2D = attr_base(key="polyline2d")
+    beizer_point_types: str = attr(is_dynamic=True, key=camel)
```

Comment:
`is_dynamic=True` should be replaced by `default=False`
Take `IsTrackingMixin` as a reference: https://github.com/linjiX/tensorbay-python-sdk/blob/f1d6551c8cfa4d8cafff73548464a25ec34cccba/tensorbay/label/supports.py#L285